### ISSUE TITLE
Add support for Href Links

### ISSUE THREAD

**Issue by ipoweb** [jackwanders/GalleryView#25]
It would be great if we can add href links as

<ul class="mgGallery">
<li><a href="#"><img src="img1.jpg"></a></li>
<li><a href="#"><img src="img1.jpg"></a></li>
<li><a href="#"><img src="img1.jpg"></a></li>
</ul>

**Comment by joeletherton** [comment]
I've just written an add on for it and will be creating a fork/submitting it for feedback once my testing is done.

**Comment by ipoweb** [comment]
i'm trying to implant lightbox or colorbox but very hard, i'm waiting your addon, if you want i test it, i'm ready :)

**Comment by joeletherton** [comment]
I've posted my code to a new fork if you want to check it out. Relevant changes are in the CHANGELOG.txt. The only changed file is the jquery.galleryview-3.0-dev.js file.

https://github.com/joeletherton/GalleryView/tree/592f6213539aef4118b8183d84b4b53dff7d97c3

**Comment by greg01** [comment]
A really great idea and a great feature. I istalliert it and it works.

But. Unfortunately, not every image can be equipped with its own link. When switching to the second image, works just the first link.
Am I doing something wrong, or work no more links with more pictures?

**Comment by joeletherton** [comment]
I've implemented it on my own site and it doesn't have this problem. I'll look at the checked in source to make sure that I didn't miss a change or something. The actual click did have some quirks at first.

http://www.joeletherton.com/Desktops

**Comment by joeletherton** [comment]
@greg01: I performed a file compare on them and the only differences between them is whitespace (tabs, spaces, etc).  Do you have your implementation in the public space anywhere or can you mail me the end result source html? joel.etherton@gmail.com

**Comment by greg01** [comment]
sure, here you need to login first.

www.flavor-selection.de
Emai: joel.etherton@gmail.com
Pass: test

Website is in development

**Comment by joeletherton** [comment]
@greg01: I viewed the .js source of the file you included and it looks like you have the version before I made the "undelegate" change. If you set the property for "link_newwindow" to be true, you'll probably notice that each time you switch images the actions stack. You can let the frames scroll through all the images and it'll open 3 windows at the same time. If you get the most recent version it shouldn't do this.

**Comment by greg01** [comment]
I think I have the latest version. I have only two days, the gallery view downloaded.

The error appears when you described by switched on link_newwindow: not true. The window opens only once.

have you any other ideas?

nice greetings from Hamburg

**Comment by joeletherton** [comment]
@greg01: I don't think you do. I compared your version with the one posted here, and it's missing a comment and the "undelegate" call located on lines 934-937. I looked into the possibility that the automatic frame cycling was doing it, but the version I have on my site works even with that.

**Comment by greg01** [comment]
I have now simply taken from your homepage and it works.
I must thank you. I've spent the last 48 hours trying to link to individual pictures. Thank you so much for your time and a great compliment to this wonderful plugin.

Greetings from Hamburg, Germany

P.S. Is a random function of the images planned?

**Comment by joeletherton** [comment]
@greg01: I added a randomness option and functionality. There's a new changelog and version in my fork (I haven't implemented this on my website so that will not have the new feature). Let me know if you have any problems with it.

**Comment by regalsingh** [comment]
I have wasted 12 hours to integrated link, But did not get any idea.
Now read your comment, its so helpful, thank you very much.

**Comment by joeletherton** [comment]
Glad I could  help.

**Comment by lathomas64** [comment]
I was having issues with the first link too but looked at your example and saw clickable: 'all' in your options and added that and it works, thanks!

**Comment by CapadY** [comment]
At the sliderpage I've added a listbox.

Slider is working fine , photonames are from a database and photos are shown well.
Listbox is filled with persons names

When the listbox is changed (onChange) I want to view a certain picture, the picture of that person
Is there somebody who can tell how to do this ?
